book/MCDM Productions; Arcadia Issue 8.json

```
@@ -0,0 +1,3593 @@
+                        {
+                            "type": "entries",
+                            "name": "Legend of the Wicked Witch",
+                            "page": 3,
+                            "entries": [
+                                "The wise woman in the woods with a gleam of mischief in her eye. The ancient crone who roams the streets, talking to alleycats. The kindly widow with the frightening smile. Hags embody humanity’s fears of manipulation, getting lost, and, worst of all, growing old. Solitary and crafty, hags typically keep only animal familiars as companions. On occasion, however, hags come together in groups of three or more to form covens, greatly increasing their power and spellcasting ability. As rife with disagreement as they are affection, hags in a coven constantly vie for control over each other.",
+                                "All hags share a few key commonalities, regardless of their variety. Most come from the Feywild and possess the faerie fixation with deals. Hags always twist the odds of these pacts in their favor. Rare is the hag without a grand plan, and rarer still is one that resists the natural temptation to prey upon human nature, bending mortals’ worst impulses for the hag’s amusement. Green hags particularly love to toy with mortal emotions. Night hags are partial to terrifying mortals to death, and sea hags relish destruction in all forms.",
+                                "The subclasses introduced in this article were developed around the legend of the hag, though any fairy-tale trickster could be used in their place. Perhaps a bard is inspired by the Big Bad Wolf, maybe a warlock seeks out Reynard the Fox, or perchance a sorcerer is descended from dreaded Queen Mab. The hag’s legend looms large, but the world of faerie is vast and rife with tales to explore."
+                            ]
+                        },
+                        {
+                            "type": "entries",
+                            "name": "Warlock Hag Patron",
+                            "page": 3,
+                            "entries": [
+                                {
+                                    "type": "image",
+                                    "href": {
+                                        "type": "external",
+                                        "url": "https://i.imgur.com/HaIaJM7.jpg"
+                                    },
+                                    "title": "Optional Title"
```

I doubt the title of that image is `Optional Title` 😛
```suggestion
                                    }
```